prov/sockets/src/sock_cntr.c

```diff
@@ -420,7 +421,11 @@ static int sock_cntr_verify_attr(struct fi_cntr_attr *attr)
 	switch (attr->wait_obj) {
 	case FI_WAIT_NONE:
 	case FI_WAIT_UNSPEC:
+#ifndef _WIN32
 	case FI_WAIT_MUTEX_COND:
+#else /* _WIN32 */
+	case FI_WAIT_CRITSEC_COND:
+#endif /* _WIN32 */
 	case FI_WAIT_SET:
 	case FI_WAIT_FD:
```

Comment:
This segment needs more work.  Windows should not support FI_WAIT_MUTEX_COND or FI_WAIT_FD.  Linux should not support FI_WAIT_CRITSEC_COND.  Yes, this means figuring out to to avoid peppering ifdefs throughout the code, but this is part of the cost of adding native windows support.